src/common/objects/Player.hpp

```diff
@@ -69,6 +77,7 @@ class Player : public GameObject {
     bool m_ungroup = false;                    // If player wants to ungroup.
     ResourceType m_intent = ResourceType::RED; // Resource player intends to collect. This is
                                                // communicated to other players.
+    WinCondition m_win_condition;
```

Comment:
`s/m_win_condition/m_winCondition`
member variables use the style `m_camelCase`